src/main/java/com/raisetech/raisetechtask/Controller/UserController.java

```diff
@@ -0,0 +1,40 @@
+package com.raisetech.raisetechtask.controller;
+
+import com.raisetech.raisetechtask.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class UserController {
+
+    //ユーザー情報サービス
+    @Autowired
+    private UserService userService;
+
+    //ユーザー情報一覧
+    @GetMapping("/users")
+    public List<UserResponse> getAll() {
+
+        return userService.findAll().stream().map(UserResponse::new).toList();
+    }
+
+    @GetMapping("/user/{id}")
```

Comment:
```suggestion
    @GetMapping("/users/{id}")
```
の方が良いです。
REST API URL設計などで調べていただくとこのようなURLで設計することを推奨しているはず。

Reply:
なるほどです！
複数形に戻しました！

[938f3ce](https://github.com/HidenoriMurai/raisetech-task9/pull/1/commits/938f3ceca4e1eb732ed7c696b1154db2cda8a177)